pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -902,6 +911,27 @@ func determineEnabledAPIGroups(discoveryClient discovery.AggregatedDiscoveryInte
 	return apiGroups, nil
 }
 
+// getFeatureGatesAPIServerMetrics extracts enabled feature gates from the API server metrics endpoint.
+// It returns a list of feature gate names that are enabled.
+func getFeatureGatesAPIServerMetrics(ctx context.Context, clientConfig *clientconfigv1.Clientset) ([]string, error) {
+	rsp, err := clientConfig.RESTClient().Get().AbsPath("/metrics").Do(ctx).Raw()
+	if err != nil {
+		return nil, err
+	}
+	featureGates := sets.NewString()
+	lines := strings.Split(string(rsp), "\n")
+	for _, line := range lines {
+		if strings.HasPrefix(line, "kubernetes_feature_enabled{") && strings.HasSuffix(line, "} 1") {
+			re := regexp.MustCompile(`kubernetes_feature_enabled\{name="([^"]+)".*\} 1`)
+			matches := re.FindStringSubmatch(line)
+			if len(matches) == 2 {
+				featureGates.Insert(matches[1])
+			}
+		}
+	}
+	return featureGates.List(), nil
```

Comment:
I'd appreciate if there was some log to note what feature gate comes from kube-apiserver metrics
@stbenjam what level would be the right one?

Reply:
I'd be fine with an `Info`-level log line for both this and the OCP side listing the gates from each